Automattic/woocommerce-payments
# Add userscript for WCPay Live Branch

Fixes https://github.com/Automattic/woocommerce-payments-meta/issues/92

TODO for myself: 

- [x] Finish the user script.
- [x] Update the comment body in [pr-build-live-branch.yml](https://github.com/Automattic/woocommerce-payments/blob/033c3a5b6bfe0e11f4c1522073d8cc943f1049f5/.github/workflows/pr-build-live-branch.yml#L116-L125)
- [x] Write a P2 post internally. 

#### Changes proposed in this Pull Request

- Add a user script like [Jetpack](https://github.com/Automattic/jetpack/tree/trunk/tools/jetpack-live-branches) and [WooCommerce core](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-beta-tester/userscripts). 
- Update the comment in the pr-build-live-branch.yml workflow to give more ways and instructions.









#### Testing instructions

#### Testing the user script 

- Check this README.md file https://github.com/Automattic/woocommerce-payments/tree/add/jn-userscript-for-live-branch/bin/wcpay-live-branches - when this PR is merged, it will be live at https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches
- Follow up the README.md file but the correct link for now would be https://github.com/Automattic/woocommerce-payments/tree/add/jn-userscript-for-live-branch/bin/wcpay-live-branches
- Refresh this pull request page, and see something like the following in this PR description, select some options and spin up a site :rocket: 

![Markup on 2023-03-21 at 11:40:01](https://user-images.githubusercontent.com/10045087/226525271-e75bb729-4fc1-4792-9989-ef736283fc01.png)

#### Check the comment and Jetpack Beta 

- Check the PR comment https://github.com/Automattic/woocommerce-payments/pull/5801#issuecomment-1473294235
- May test the links there and try Jetpack Beta. 





*

-------------------


**Post merge**

NO NEED.
